src/renderer/shared/reducers/toastsReducer.js

```diff
@@ -0,0 +1,30 @@
+import uuid from 'uuid/v1';
+import { takeRight, filter } from 'lodash';
+
+import { ENQUEUE, DEQUEUE } from '../actions/toastsActions';
+
+const initialState = [];
+
+const MAX_TOASTS = 3;
```

Comment:
Should we move this out to `./**/values` folder - a matter of centralizing all "configurable" things 🙈 What do you think?